src/components/pages/questions/[question_id]/QuestionCard.tsx

```diff
@@ -36,11 +38,49 @@ export default function QuestionCard({ question, href }: QuestionCardProps) {
     void fetchTags()
   }, [])
 
+  // Function to handle profile navigation
+  const handleProfileClick = () => {
+    if (!href && question.asker_info) {
+      router.push(`/profiles/${question.asker_info.username}`)
+    }
+  }
+
+  // Function to handle community navigation
+  const handleCommunityClick = () => {
+    if (!href && question.related_community_info) {
+      router.push(`/communities/${question.related_community_info.title}`)
+    }
+  }
+
   const QuestionCardContent = (
     <Card
       className={`w-full ${href && 'cursor-pointer transition-transform duration-200 hover:scale-105 hover:shadow-lg'}`}
     >
       <CardHeader>
+        {!href && question.related_community_info ? (
+          <div className="flex items-center space-x-3 rounded-t-lg pb-2">
```

Comment:
could we use the same styling for clicking on `user` and `community` (the community looks more like a link, but user looks more like a button)